rpc_jobs/rpc_multi_node_aio.yml

```diff
@@ -0,0 +1,143 @@
+- project:
+    name: 'RPC-Multi-Node-AIO-Jobs'
+    # Note: branch is the branch for periodics to build
+    #       branches is the branch pattern to match for PR Jobs.
+    series:
+      - liberty:
+          branch: liberty-12.2
+          OPENSTACK_ANSIBLE_BRANCH: "stable/liberty"
+      - mitaka:
+          branch: mitaka-13.1
+          OPENSTACK_ANSIBLE_BRANCH: "stable/mitaka"
+      - newton:
+          branch: newton-14.0
+          OPENSTACK_ANSIBLE_BRANCH: "stable/newton"
+      - master:
+          branch: master
+          branches: "master"
+          OPENSTACK_ANSIBLE_BRANCH: "stable/newton"
+    context:
+      - trusty:
+          DEFAULT_IMAGE: "14.04.5"
+      - xenial:
+          DEFAULT_IMAGE: "16.04.2"
+
+    # NOTE: Hugh tested this and found that ztrigger overrides series and
+    #       trigger doesn't, which is odd because both trigger and ztrigger
+    #       sort after series.
+    ztrigger:
+      - periodic
+    exclude:
+      # Xenial builds are run for newton and above.
+      - series: liberty
+        context: xenial
+      - series: mitaka
+        context: xenial
+    jobs:
+      - 'RPC-Multi-Node-AIO_{series}-{context}-{ztrigger}'
+
+- job-template:
+    # DEFAULTS
+    branch: master
```

Comment:
Yeah, I believe these aren't needed as I just tested commenting them out and the file still test correctly.  I think we'd only need to set a default if we have a matrix where the variable doesn't necessarily get set in each combination, or if it's referenced in the job template (ie. NUM_TO_KEEP).  I'll get those two vars removed from the template.